src/components/book/bookContentCard.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component, Fragment } from "react";
+
+//MUI Core
+import Card from "@material-ui/core/Card";
+import CardActionArea from "@material-ui/core/CardActionArea";
+import CardActions from "@material-ui/core/CardActions";
+import CardContent from "@material-ui/core/CardContent";
+import CardMedia from "@material-ui/core/CardMedia";
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import withStyles from "@material-ui/core/styles/withStyles";
+// MUI Icon
+import ShareIcon from "@material-ui/icons/Share";
+
+// Component
+import BookContentCardDetail from "../book/bookContentCardDetail";
+
+const styles = {
+  root: {
+    maxWidth: 280,
+    marginBottom: "2em",
+    marginLeft: "0.5em",
+  },
+  media: {
+    width: "14em",
+    height: 280,
+    backgroundSize: "cover",
+  },
+  content: {
+    width: "14em",
+    height: 200,
+  },
+  cardActions: {
+    display: "flex",
+    justifyContent: "right",
+  },
+  button: {
+    color: "#1c2a48",
+    "&:nth-child(2)": {
+      marginLeft: "-1em",
+    },
+  },
+  title: {
+    marginTop: "0.5em",
+    wordWrap: "break-word",
+    fontSize: "1rem",
+    // width: "12em"
+  },
+  desc: {
+    marginTop: "1em",
+    wordWrap: "break-word",
+  },
+};
+
+class BookContentCard extends Component {
+  render() {
+    const { bookImage, author, title, desc, userImage } = this.props.book;
+    const { classes } = this.props;
+    return (
+      <Fragment>
+        <Card className={classes.root}>
+          <CardActionArea>
+            <CardMedia
+              className={classes.media}
+              image={bookImage}
+              title={title}
+            />
+            <CardContent className={classes.content}>
+              {/* <Typography gutterBottom variant="subtitle1" component="span" color="primary">
+                {author}
+              </Typography> */}
+              <Typography
+                className={classes.title}
+                gutterBottom
+                variant="h6"
+                component="h6"
+                align="center"
+              >
+                {title.slice(0, 25)}
```

Comment:
Martin 2 comments:
1) Can we name component files with Pascal Case?
I.e. this file should be BookContentCard( Pascal Case), not bookContentCard ( Camel Case).
This follows the naming scheme of the other components! (if you are making a component under "page" we can have camel case but this is under "components")

2) In [PT-150](https://proudtale.atlassian.net/browse/PT-150?atlOrigin=eyJpIjoiZTgwZDcyNjUyMGFkNGVmZTkxMGMyYjFmNTg5YzgyNjgiLCJwIjoiaiJ9) we decided that thumbnails should be about 14 characters in length :). I made a function in utils/helper.js to match PT-150's expectations so please take a look at that!

Reply:
I could be wrong, but typically, I believe camelCase is used for Javascript. You do bring a good point on it being in "components."

Reply:
Okay. I will rename component files with Pascal Case? And I will use function in utils to slice thumbnail.

Reply:
@mdc2tanu yes but also https://www.quora.com/Why-do-React-components-use-PascalCase-for-filenames

Reply:
> Martin 2 comments:
> 
>     1. Can we name component files with Pascal Case?
>        I.e. this file should be BookContentCard( Pascal Case), not bookContentCard ( Camel Case).
>        This follows the naming scheme of the other components! (if you are making a component under "page" we can have camel case but this is under "components")
> 
>     2. In [PT-150](https://proudtale.atlassian.net/browse/PT-150?atlOrigin=eyJpIjoiZTgwZDcyNjUyMGFkNGVmZTkxMGMyYjFmNTg5YzgyNjgiLCJwIjoiaiJ9) we decided that thumbnails should be about 14 characters in length :). I made a function in utils/helper.js to match PT-150's expectations so please take a look at that! 
=> **I completed 2**